src/SIL.Harmony.Sample/Changes/TagWordChange.cs

```diff
@@ -0,0 +1,35 @@
+﻿using SIL.Harmony.Changes;
+using SIL.Harmony.Entities;
+using SIL.Harmony.Sample.Models;
+
+namespace SIL.Harmony.Sample.Changes;
+
+public class TagWordChange(WordTag wordTag) : CreateChange<WordTag>(wordTag.Id == Guid.Empty ? Guid.NewGuid() : wordTag.Id), ISelfNamedType<TagWordChange>
+{
+    public Guid WordId { get; } = wordTag.WordId;
+    public Guid TagId { get; } = wordTag.TagId;
+
+    public async override ValueTask<WordTag> NewEntity(Commit commit, IChangeContext context)
+    {
+        var isDuplicate = await IsDuplicate(context);
+        return new WordTag()
+        {
+            Id = EntityId,
+            WordId = WordId,
+            TagId = TagId,
+            DeletedAt = isDuplicate ? DateTimeOffset.UtcNow : null
```

Comment:
Should be using the commit date, otherwise different users would see different deleted dates